src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss

```diff
@@ -49,44 +48,34 @@ $secondErrorColor : #b42626;
 
     .buttons {
       display: flex;
-      align-items: flex-end;
-
-      label {
-        margin: 0;
-        padding-top: 16px;
-      }
+      align-items: center;
 
       .primary-global-button,
       .secondary-global-button {
-        border: 1px solid var(--primary-green);
+        margin: 0 8px;
+        padding: 8px 24px;
         box-sizing: border-box;
-        border-radius: 5px;
+        border-radius: 4px;
         font-style: normal;
-        font-weight: 600;
+        font-weight: bold;
         font-size: 16px;
-        height: 48px;
-        line-height: 16px;
+        line-height: 24px;
       }
 
       .primary-global-button {
         background: #fff;
```

Comment:
maybe better to create a variable for this color (such as --primary-white)?

Reply:
Added